src/components/smart_objects/src/object_schema_item.cc

```diff
@@ -78,23 +82,8 @@ SMember::SMember(const ISchemaItemPtr SchemaItem,
 bool SMember::CheckHistoryFieldVersion(
     const utils::SemanticVersion& MessageVersion) const {
   if (MessageVersion.isValid()) {
-    if (mSince != boost::none) {
```

Comment:
Why was this removed?

Reply:
@jacobkeeler please check 165604adbbcc033bb1fa837024a58ee8e5c72592